packages/core/src/config.ts

```diff
@@ -536,6 +541,23 @@ const composeFormatConfig = (format: Format): RsbuildConfig => {
           },
         },
       };
+    case 'mf':
+      return {
+        tools: {
+          rspack: {
+            output: {
+              uniqueName: pkgJson.name as string,
+            },
+            // TODO when we provide dev mode for rslib mf format, this should be modified to as the same with config.mode
```

Comment:
What is it mean?

Reply:
To ensure process.env.NODE_ENV variable is correct